Packs/FeedMISP/Integrations/FeedMISP/FeedMISP_test.py

```diff
@@ -562,3 +563,23 @@ def test_build_indicators_from_galaxies_attack_type():
     assert galaxy_indicators['type'] == 'Attack Pattern'
     assert galaxy_indicators['service'] == 'MISP'
     assert galaxy_indicators['Reputation'] == 'High'
+
+
+def test_build_indicators_with_hostname():
+    """
+    Given:
+        - A response from the API which contains an indicator of type hostname.
+    When:
+        - build_indicators executed.
+    Then:
+        - Successfully creates indicator and relationship.
+    """
+    from FeedMISP import build_indicators
+    client = Mock()
+    response = {'response': {'Attribute': [{'type': 'hostname',
+                                            'value': {'value': '1.1.1.1'},
```

Comment:
```suggestion
                                            'value': {'value': '1.1.1.1'},
```
                                            'value': {'value': 'www.test.com'},